src/CopyrighFixer/CopyrighFixer/signer.h

```diff
@@ -27,8 +28,16 @@ class CopyrighFixer_EXPORT Signer {
      */
     bool checkSign(const Config &objConf);
 
+    /**
+     * @brief searchFileByExt The method that searches for files with the desired extension.
+     * @param extension This is the file extension to search.
+     * @return Returns a pointer to the FileManager of the found extension.
+     */
     IFileManager *searchFileByExt(const QString &extension);
 
+private:
+    QList<IFileManager*> FileManager;
```

Comment:
all private field begin from _ 
```suggestion
    QList<IFileManager*> _fileManager;
```